### PR TITLE
COH-22271 Push CE snapshots to maven central

### PR DESCRIPTION
**Pull request by ghillert**
- Remove Nexus repo from `synoki.xml`
- Update `deploy-snapshots.yaml`

### CODE REVIEW

File: .github/maven/synoki.xml [hunk 1]
```diff
@@ -2,46 +2,12 @@
 
   <servers>
     <server>
-      <id>synoki-nexus</id>
+      <id>ossrh</id>
       <username>${env.MAVEN_USER}</username>
       <password>${env.MAVEN_PASSWORD}</password>
```
Comment (aseovic) [inline .github/maven/synoki.xml:7]: Do we still need this file at all? If we do, can you rename it to `settings.xml`?

File: .github/workflows/deploy-snapshots.yaml [hunk 2]
```diff
@@ -31,4 +31,4 @@ jobs:
       env:
         MAVEN_USER: ${{ secrets.MavenUser }}
         MAVEN_PASSWORD: ${{ secrets.MavenPassword }}
-      run: mvn -B deploy --file prj/pom.xml -DskipTests -s .github/maven/synoki.xml -DaltDeploymentRepository=synoki-nexus::default::https://nexus.synoki.io/repository/maven-snapshots/
+      run: mvn -B deploy --file prj/pom.xml -DskipTests -s .github/maven/synoki.xml
```
Comment (aseovic) [inline .github/workflows/deploy-snapshots.yaml:34]: Again, if we need settings file, let's name it `settings.xml` instead of `synoki.xml`

**Review by aseovic — CHANGES_REQUESTED** [review]
A few minor changes are still needed, but otherwise looks good. No need to review it again.

File: .github/maven/synoki.xml (hunk 1 above)
Comment (ghillert) [reply .github/maven/synoki.xml]: Renamed

**Review by aseovic — APPROVED** [review]

**Pull request closed without merging**